minimalDDA.mzML

```diff
@@ -0,0 +1,391 @@
+              <binary>AAAAAAAAAEAAAAAAAAAIQA==</binary>
+            </binaryDataArray>
+            <binaryDataArray encodedLength="12">
+              <cvParam cvRef="MS" accession="MS:1000521" name="32-bit float" value=""/>
+              <cvParam cvRef="MS" accession="MS:1000576" name="no compression" value=""/>
+              <cvParam cvRef="MS" accession="MS:1000515" name="intensity array" value="" unitCvRef="MS" unitAccession="MS:1000131" unitName="number of detector counts"/>
+              <binary>AIAiRAAAAAA=</binary>
+            </binaryDataArray>
+          </binaryDataArrayList>
+        </spectrum>
+      </spectrumList>
+      <chromatogramList count="1" defaultDataProcessingRef="pwiz_Reader_Waters_conversion">
+        <chromatogram index="0" id="TIC" defaultArrayLength="9">
+          <cvParam cvRef="MS" accession="MS:1000235" name="total ion current chromatogram" value=""/>
+          <binaryDataArrayList count="3">
+            <binaryDataArray encodedLength="96">
+              <cvParam cvRef="MS" accession="MS:1000523" name="64-bit float" value=""/>
+              <cvParam cvRef="MS" accession="MS:1000576" name="no compression" value=""/>
+              <cvParam cvRef="MS" accession="MS:1000595" name="time array" value="" unitCvRef="UO" unitAccession="UO:0000031" unitName="minute"/>
+              <binary>AAAAAAAA8D8AAAAAAAAAQAAAAEAzMwNAAAAAAAAABEAAAACgmZkFQAAAAGBmZgZAAAAAAAAACEAAAABAMzMLQAAAAAAAAAxA</binary>
+            </binaryDataArray>
+            <binaryDataArray encodedLength="48">
+              <cvParam cvRef="MS" accession="MS:1000521" name="32-bit float" value=""/>
+              <cvParam cvRef="MS" accession="MS:1000576" name="no compression" value=""/>
+              <cvParam cvRef="MS" accession="MS:1000515" name="intensity array" value="" unitCvRef="MS" unitAccession="MS:1000131" unitName="number of detector counts"/>
+              <binary>AEBORQBATkUAABZEAABIQgCAIkQAAEhCAEBORQCAIkQAAEhC</binary>
+            </binaryDataArray>
+            <binaryDataArray arrayLength="9" encodedLength="96">
+              <cvParam cvRef="MS" accession="MS:1000522" name="64-bit integer" value=""/>
+              <cvParam cvRef="MS" accession="MS:1000576" name="no compression" value=""/>
+              <cvParam cvRef="MS" accession="MS:1000786" name="non-standard data array" value="function" unitCvRef="UO" unitAccession="UO:0000186" unitName="dimensionless unit"/>
+              <binary>AQAAAAAAAAABAAAAAAAAAAIAAAAAAAAAAwAAAAAAAAACAAAAAAAAAAMAAAAAAAAAAQAAAAAAAAACAAAAAAAAAAMAAAAAAAAA</binary>
+            </binaryDataArray>
+          </binaryDataArrayList>
+        </chromatogram>
+      </chromatogramList>
+    </run>
+  </mzML>
+  <indexList count="2">
+    <index name="spectrum">
+      <offset idRef="function=1 process=0 scan=1">6412</offset>
+      <offset idRef="function=1 process=0 scan=2">9032</offset>
+      <offset idRef="function=2 process=0 scan=1 merged=1-2">11652</offset>
```

Comment:
Oops, actually the id should start with merged=, not end with it. And it should be equal to the index of the scan (so it doesn't add information, but that's the convention in the mzML standard). So the 1-2 or 1-3 part should go in the scan= field.

Reply:
I tried putting the range in `scan=` but it appeared to break SeeMS, and I was also worried about other consuming applications which expected a single number too.

Do we need to fix up SeeMS too?

Reply:
I considered that might be an issue but didn't try it. I thought maybe we should change it to scans= and now I think we have to. The merged= at the beginning should have signified to SeeMS that it wasn't a regular nativeID it could parse though...

Reply:
Ah ok, maybe I didn't have the `merged=` at the beginning.

So what should the id be, exactly? I'm guessing

```
merged=4 scans=4-7 function=2 process=0
```

Reply:
Ok, I fixed that, and SeeMS seems *reasonably* happy. It will load the file, but doesn't show anything in the Function, Process or Scan column. You can see all the info in the id column.

Reply:
That is expected. Adding extra fields after merged= is a fairly new development and not really the established convention.